### ISSUE TITLE
Non-comp Flow | Reviews landing page

### ISSUE THREAD

**Issue by shanear** [department-of-veterans-affairs/caseflow#7990]
Compensation and Pension use VBMS to process HLRs and SCs. Other business lines do not, thus we should not create EPs for those. Instead we will use the DB objects directly in the Caseflow non-comp UI. This is the page where those HLRs and SCs will appear.

## Acceptance Criteria
- When a SC or HLR is created and has a benefit type other than Compensation or Pension, it should appear as a task on the business line review page associated with that benefit type.
- Visiting the `/decision_reviews/[business-line-slug]` url should show the review page associated with the key for that business line (i.e. `/decision_reviews/vocational_rehab_and_employment` should show the Vocational Rehab and Employment reviews page)
- "Days Waiting" column should be the number of days since the task was added to the queue (which would be the date the decision review intake was completed)
- The "Intake New Form" link should route the user to the intake flow `/intake`

## Notes
Out of scope for this story:
- The "Completed" tab
- The search bar
- The link for clicking on an individual task
- "Board Grant" tasks
- order of the tasks
- pagination

Full non-comp flow: https://va-gov.invisionapp.com/share/PFPCPJ7MEGC

## Mocks

![returning disposition main](https://user-images.githubusercontent.com/11700545/49621592-01b91300-f97b-11e8-98db-00f8cf0702cd.png)

## Technical Notes
While issues typically do not contain details about technical implementation, there's enough open questions that could have long term implications that we've discussed a rough approach:

1. We will build this noncomp flow into a new Redux app, separate from Intake/IntakeEdit/Queue
2. We will reuse whatever Queue frontend code we can, refactoring to make generic if necessary. Look to reuse reducers, UI components, logic. See TaskTable as an example
3. We will reuse Queue backend code - Organizations can map to business lines, Task can be used directly with some modifications to support a polymorphic appeal/HLR/SC association (currently only appeal)

There will be potentially be many different kinds of tasks on this page.

See additional notes here: https://github.com/department-of-veterans-affairs/appeals-design-research/issues/1081

There's a [sample PR](https://github.com/department-of-veterans-affairs/caseflow/pull/8111/files) that Mark put together that shows how this might be built into Queue - a good reference to use for what components can be shared, though we want to build this into a new Redux app.

References:
* [models/tasks/generic_task.rb](https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/models/tasks/generic_task.rb)
* [controllers/tasks_controller.rb](https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/controllers/tasks_controller.rb)
* [Generic task queue tech spec](https://github.com/department-of-veterans-affairs/caseflow/issues/6573)
* [Example of task implementation](https://github.com/department-of-veterans-affairs/caseflow/pull/7281/files)

Notes from @mdbenjam:
> Specifically look at `app/models/organizations/quality_review.rb` and `app/models/tasks/quality_review_task.rb`. Neither of these is strictly necessary to spin up a generic queue. We often add an organization directly to the organization table, that gives them a URL to go to in order to see all of the tasks assigned to that organization. We can assign any type of task to the organization. We have a number of different types of tasks. They all inherit from `Task.rb`. We’re building GenericTasks to have a set of basic work flow management actions. i.e. assigning sub tasks to other users. For rolling out queues to other teams outside of the board we’ll probably need to make some new task types that will have specific logic upon completion to do specific things. i.e. in a DTA error assigned to NCA (not sure that’s a real scenario) completing the task assigned to them might need to re-open the task associated with the appeal at the board.

requires #7985

**Comment by carolaponce** [comment]
Updated comp

Changing the Intake form button to a main button style.
Changing text to: Review each issue and select a disposition.
![returning disposition main](https://user-images.githubusercontent.com/11700545/49259213-5396f180-f3ed-11e8-9016-09e3f24897e0.png)

**Comment by kencheeto** [comment]
@carolaponce thanks - I've edited the description to include the latest mock

**Comment by carolaponce** [comment]
Final Mock

![returning disposition main](https://user-images.githubusercontent.com/11700545/49621592-01b91300-f97b-11e8-98db-00f8cf0702cd.png)

**Comment by pkarman** [comment]
@jiahuang I believe once #8468 is merged, we can move this to validation?

**Comment by steve-gov** [comment]
Per a Slack thread and in person conversation, @jiahuang may have to manually establish an organization et. al. so this can be UATed.

**Comment by steve-gov** [comment]
Intook both a SC and HLR for `123458314` in the NCA LOB. Neither on are listed on the reviews landing page.

<img width="818" alt="screen shot 2019-01-03 at 4 44 11 pm" src="https://user-images.githubusercontent.com/25468689/50668890-d7a54180-0f76-11e9-9fe0-fd88573f4a11.png">

<img width="979" alt="screen shot 2019-01-03 at 4 43 59 pm" src="https://user-images.githubusercontent.com/25468689/50668900-db38c880-0f76-11e9-8403-9a8db889daeb.png">

**Comment by pkarman** [comment]
I assume "In progress tasks" was also empty?

**Comment by steve-gov** [comment]
It was empty at the time I added the validation-failed tag, but immediately thereafter Jia made some updates that caused the HLR to appear in the "In progress tasks" tab.

**Comment by steve-gov** [comment]
Successfully "intook" three appeals for NCA, and each one immediately showed up on the In Progress tab.
